app/api/serializers.py

```diff
@@ -164,6 +166,17 @@ def to_internal_value(self, data):
 class WidgetSerializer(serializers.ModelSerializer):
     meta_data = serializers.JSONField(source="metadata", required=False)
     dir = serializers.CharField(read_only=True)
+    creator = serializers.SerializerMethodField()
+
+    def get_creator(self, widget):
+        """
+        Checks if the widget uses the default creator - if so, returns the path for that
+        Otherwise, returns the value the widget has set.
+        """
+        if widget.creator == "default" or widget.creator == "":
+            return settings.URLS["STATIC_CROSSDOMAIN"] + "default-creator/creator.html"
```

Comment:
We're assuming that `STATIC_CROSSDOMAIN` has a trailing slash here, so we just want to make sure that the path is always going to contain one, per #111.